### PR TITLE
Earnfi integration

### PR DESCRIPTION
**Pull request by LeonmanRolls**

### CODE REVIEW

File: src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx [hunk 1]
```diff
@@ -164,71 +195,199 @@ const Miscellaneous = ({
...
+    } catch (e) {
+      setError('email', {
+        type: 'submitError',
+        message: t('tabs.more.misc.earnfi.submitError', { ns: 'profile' }),
+      })
+    }
+    setTimeout(() => {
+      clearErrors('email')
+    }, 3000)
+  }
```
Comment (TateB) [inline src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx:250]: can this be a `useMutate` hook?

File: src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx [hunk 2]
```diff
@@ -164,71 +195,199 @@ const Miscellaneous = ({
...
+                    }}
+                  >
+                    {t('action.close')}
+                  </Button>
+                }
+              />
+            </>
+          ))
+          .exhaustive()}
+      </Dialog>
```
Comment (TateB) [inline src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx:326]: probs want to pull this out into a separate component

File: src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx [hunk 3]
```diff
@@ -164,71 +195,199 @@ const Miscellaneous = ({
...
+              </Typography>
+              <Spacer $height="3" />
+              <Input
+                type="email"
+                id="email"
+                label={t('action.enterEmail')}
+                {...register('email', {
+                  required: t('errors.emailRequired'),
+                  pattern: {
+                    value: emailRegex,
```
Comment (TateB) [inline src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx:279]: can native email validation be used? (https://developer.mozilla.org/en-US/docs/Learn/Forms/HTML5_input_types#email_address_field)

**Review by TateB — CHANGES_REQUESTED** [review]

File: src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx (hunk 2 above)
Comment (storywithoutend) [reply src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx]: done

File: src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx (hunk 1 above)
Comment (storywithoutend) [reply src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx]: done

**Review by TateB — APPROVED** [review]

**Pull request merged**